databroker/assets/sqlite.py

```diff
@@ -137,6 +192,62 @@ def reconnect(self):
                                    "have expected schema. Expected "
                                    "tables: {}; found tables: {}".format(
                                        self._fp, EXPECTED_TABLES, tables))
+        cur_cursor = None
+        while not self.__shutdown_event.is_set():
+            finished_event = None
+            try:
+                item = self.__request_queue.get(timeout=0.5)
+            except queue.Empty:
+                # Check whether we are shutting down (and should therefore
+                # terminate this loop) and then resume waiting on the
+                # queue.
+                continue
+            try:
+                finished_event, name, args, kwargs, ret = item
+            except ValueError:
+                print("did not get the right number of values in {item}".
+                      format(item=item))
+                continue
+            try:
+                print(f'handling {name}')
```

Comment:
Stray print.

Reply:
I pushed a commit to fix this since I have checked it out anyway. :-)

Reply:
Thanks.